<!DOCTYPE html>
<html lang="" dir="ltr">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>Spatially explicit predictions of food web structure from
regional level data</title>
    <script src="https://hypothes.is/embed.js" async></script>
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
        integrity="sha256-l85OmPOjvil/SOvVt3HnSSjzF1TUMyT9eV0c2BzEGzU=" crossorigin="anonymous" />
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap');
        @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,500;1,400;1,500&display=swap');

        :root {
            --citation: #1976D2;
            --structure: hsl(210, 61%, 43%);

            --light: #474747;
            --pale: #a5a5a5;
            --grey: #e0e0e0;
            --nav: #fdfdfd;

            --width: 820px;
            --inner-space: 1.2em;
        }

        html {
            background-color: white;
        }

        body {
            font-family: "Roboto", sans-serif;
            margin: 0 0;
            padding: 0 0;
        }

        article>p {
            line-height: 135%;
        }

        article>h1 {
            padding-top: 3em;
        }

        a {
            text-decoration: none;
            color: #000;
            border-bottom: 1px solid var(--structure);
        }

        a:hover {
            font-weight: 500;
        }

        .citation {
            color: var(--citation);
        }

        .container {
            margin: 0em auto;
            max-width: var(--width);
            padding: var(--inner-space);
        }

        header>.container {
            margin-top: 4em;
        }

        code,
        pre {
            font-family: "Roboto Mono", monospace;
        }

        pre {
            white-space: pre-wrap;
        }

        h1,
        h2,
        h3 {
            color: var(--structure);
            font-family: "Roboto Condensed", sans-serif;
            font-weight: 400;
        }

        header {
            font-family: "Roboto Condensed", sans-serif;
            z-index: 5;
            color: black;
            margin-top: 4rem;
        }

        .header--title {
            font-weight: 400;
            font-size: 2rem;
            margin-bottom: 1rem;
        }

        .header--authors>span {
            color: var(--light);
            font-weight: 400;
        }

        main {
            padding-top: 8em !important;
            padding-bottom: 8em !important;
        }

        footer {
            background-color: var(--grey);
            font-weight: 300;
        }

        .abstract {
            color: var(--light);
            margin-bottom: 5em;
        }

        figure {
            max-width: 100%;
            border: 1px solid var(--pale);
            border-radius: 2px;
            margin: 1em 0.5em;
        }

        figure>img {
            max-width: 100%;
            display: block;
            margin: 0em auto;
        }

        figure>figcaption {
            padding: var(--inner-space);
        }

        table {
            width: 100%;
            min-width: 100%;
            border-collapse: collapse;
        }

        table>caption {
            padding: var(--inner-space);
        }

        figcaption,
        caption {
            color: var(--light);
            font-weight: 300;
        }

        thead {
            color: var(--structure);
        }

        td,
        th {
            padding: 3px;
        }

        tr {
            border-top: 1px solid var(--structure);
            border-bottom: 1px solid var(--structure);
        }

        tbody>tr:nth-child(even) {
            background: var(--grey)
        }

        ;

        tbody {
            border-bottom: 1px solid var(--structure);
        }

        #refs {
            font-size: 90%;
            column-count: 2;
        }

        #refs>.csl-entry {
            padding-bottom: 0.2em;
            padding-top: 0.2em;
        }

        .hanging-indent>.csl-entry {
            padding-left: 15px;
            text-indent: -15px;
        }

        .artifacts {
            text-align: center;
            border-radius: 4px;
            background-color: var(--grey);
            padding: 1em;
            margin-top: 4em;
        }

        .artifacts a{
            background-color: var(--structure);
            text-decoration: none;
            margin: 0px 2em;
            color: white;
            text-transform: capitalize;
            border-radius: 4px;
            text-align: center;
            padding: 4px 4px;
            width: 7em;
            display: inline-block;
        }    </style>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.css"
        integrity="sha384-vKruj+a13U8yHIkAyGgK1J3ArTLzrFGBbBc0tDp4ad/EyewESeXE/Iv67Aj8gKZ0" crossorigin="anonymous">
    
    
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/katex.min.js"
        integrity="sha384-PwRUT/YqbnEjkZO0zZxNqcxACrXe+j766U2amXcgMg5457rve2Y7I6ZJSm2A0mS4"
        crossorigin="anonymous"></script>
    
    
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.4/dist/contrib/auto-render.min.js"
        integrity="sha384-+VBxd3r6XgURycqtZ117nYw44OOcIax56Z4dCRWbxyPt0Koah1uHoK0o4+/RRE05" crossorigin="anonymous"
        onload="renderMathInElement(document.body);"></script>

    <script>
        document.addEventListener("DOMContentLoaded", function () {
            renderMathInElement(document.body, {
                delimiters: [
                    { left: "$$", right: "$$", display: true },
                    { left: "$", right: "$", display: false }
                ]
            });
        });
    </script>
</head>

<body>
    <header>
        <div class="container">
            <div class="header--title">Spatially explicit predictions of
food web structure from regional level data</div>
            <div class="header--authors">
                <span class="header--author">
                <a href="https://orcid.org/0000-0002-2212-3584" target="_blank">
                    Gabriel Dansereau
                </a>
                <sup class="affiliation">1,2</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0003-4036-977X" target="_blank">
                    Ceres Barros
                </a>
                <sup class="affiliation">3</sup>
                </span><span class="header--author">
                <a href="https://orcid.org/0000-0002-0735-5184" target="_blank">
                    Timothée Poisot
                </a>
                <sup class="affiliation">1,2</sup>
                </span>
            </div>
        </div>
    </header>

    <main class="container">
        <section class="abstract">
            

            <dl>
                    <dt>Abstract</dt>
                <dd>Knowledge about how ecological networks vary across
            global scales is currently limited given the complexity of
            acquiring repeated spatial data for species interactions.
            Yet, recent developments of metawebs highlight efficient
            ways to first document possible interactions within regional
            species pools. Downscaling metawebs towards local network
            predictions is a promising approach to use current data to
            investigate the variation of networks across space. However,
            issues remain in how to represent the spatial variability
            and uncertainty of species interactions, especially for
            large-scale food webs. Here, we present a probabilistic
            framework to downscale a metaweb based on the Canadian
            mammal metaweb and species occurrences from global
            databases. We investigated how our approach can be used to
            represent the variability of networks and communities
            between ecoregions in Canada. Species richness and
            interactions followed a similar latitudinal gradient across
            ecoregions but simultaneously identified contrasting
            diversity hotspots. Network motifs revealed additional areas
            of variation in network structure compared to species
            richness and number of links. Our method offers the
            potential to bring global predictions down to a more
            actionable local scale, and increases the diversity of
            ecological networks that can be projected in space.</dd>
                </dl>

            <div class="artifacts">
                <a href="Dansereau_et_al_2024_spatially_explicit_draft.pdf" title="Draft" target="_blank">Draft</a>
                <a href="Dansereau_et_al_2024_spatially_explicit_preprint.pdf" title="Preprint" target="_blank">Preprint</a>
                <a href="Dansereau_et_al_2024_spatially_explicit.odt" title="OpenDocument" target="_blank">Libre Office</a>
            </div>
        </section>

        <article>
            <h1 id="introduction">Introduction</h1>
            <p>Because species interactions vary in time and space, and
            because species show high turnover over larger spatial
            extents, adequately capturing the diversity of ecological
            networks is a challenging task <span class="citation"
            data-cites="Jordano2016SamNet">(Jordano 2016)</span>. Most
            studies on food webs have previously focused on local
            networks limited in size and extent, and are rarely
            replicated in space or time <span class="citation"
            data-cites="Mestre2022DisFoo">(Mestre <em>et al.</em>
            2022)</span>. Interactions can show important variations in
            space <span class="citation"
            data-cites="Poisot2015SpeWhy Zarnetske2017IntLan">(Poisot
            <em>et al.</em> 2015; Zarnetske <em>et al.</em>
            2017)</span>, yet available network data also show important
            geographical bias by focusing sampling efforts in a few
            areas or biomes, limiting our ability to answer questions in
            many biomes and over broad spatial extents <span
            class="citation" data-cites="Poisot2021GloKno">(Poisot
            <em>et al.</em> 2021)</span>. Moreover, global monitoring of
            biotic interactions is insufficient to properly describe and
            understand how ecosystems are reacting to global change
            <span class="citation"
            data-cites="Windsor2023UsiEco">(Windsor <em>et al.</em>
            2023)</span>. Approaches to predict species interactions
            <span class="citation"
            data-cites="Desjardins-Proulx2017EcoInt Morales-Castilla2015InfBio">(<em>e.g.</em>,
            Desjardins-Proulx <em>et al.</em> 2017; Morales-Castilla
            <em>et al.</em> 2015)</span> are increasingly used as an
            alternative to determine potential interactions; they can
            handle limited data to circumvent data scarcity <span
            class="citation" data-cites="Strydom2021RoaPre">(Strydom
            <em>et al.</em> 2021)</span>, but are still rarely used to
            make explicitly spatial predictions. As a result, there have
            been repeated calls for globally distributed interaction and
            network data coupled to repeated sampling in time and space
            <span class="citation"
            data-cites="Mestre2022DisFoo Windsor2023UsiEco">(Mestre
            <em>et al.</em> 2022; Windsor <em>et al.</em> 2023)</span>,
            which will help understand the macroecological variations of
            food webs <span class="citation"
            data-cites="Baiser2019EcoRul">(Baiser <em>et al.</em>
            2019)</span>. Despite these limitations, food web ecologists
            often can infer a reasonable approximation of the network
            existing within a region. This representation, called a
            metaweb, contains all possible interactions between species
            in a given regional species pool <span class="citation"
            data-cites="Dunne2006NetStr">(Dunne 2006)</span>, and
            provides a solid foundation to develop approaches to
            estimate the structure of networks at finer spatial
            scales.</p>
            <p>When assembled by integrating different data sources (and
            potentially with additional predictive steps), the metaweb
            allows to overcome sampling limitations and to raise network
            data to a global scale. For example, <span class="citation"
            data-cites="Albouy2019MarFis">Albouy <em>et al.</em>
            (2019)</span> coupled data on fish distributions with a
            statistical model of trophic interactions to provide
            estimates of the potential food web structure at the global
            scale. Recent studies have focused on assembling metawebs
            for various taxa through literature surveys and expert
            elicitation <span class="citation"
            data-cites="Maiorano2020TetSpe">(European terrestrial
            tetrapods, Maiorano <em>et al.</em> 2020)</span> or using
            predictive tools <span class="citation"
            data-cites="Albouy2019MarFis Strydom2022FooWeb">(marine
            fishes, Albouy <em>et al.</em> 2019; Canadian mammals,
            Strydom <em>et al.</em> 2022)</span>. At a finer spatial
            scale, the local food webs <span class="citation"
            data-cites="Poisot2012DisSpe">(<em>i.e.</em>, the local
            “realization” of the metaweb when combined with species
            distributions, Poisot <em>et al.</em> 2012)</span> reflect
            local environmental conditions but still retain the signal
            of the metaweb to which they belong <span class="citation"
            data-cites="Saravia2022EcoNet">(Saravia <em>et al.</em>
            2022)</span>. Given this, <span class="citation"
            data-cites="Strydom2023GraEmb">Strydom <em>et al.</em>
            (2023)</span> defended that predicting the metaweb’s
            structure should be the core goal of predictive network
            ecology. Assuming there is a strong link between the metaweb
            and its local realizations, more accurate predictions of the
            metaweb will have the potential to bring us closer to
            producing accurate local (downscaled) predictions <span
            class="citation" data-cites="Strydom2023GraEmb">(Strydom
            <em>et al.</em> 2023)</span>. Therefore, establishing or
            predicting the metaweb should be the first target in
            communities where data about local realizations
            (<em>i.e.</em>, documented interactions at specific places)
            are lacking. Our approach differs from using interactions to
            improve predictions of species distributions, as has been
            done by recent studies <span class="citation"
            data-cites="Moens2022ImpBio Poggiato2022IntFoo Lucas2023IncBio">(Lucas
            <em>et al.</em> 2023; Moens <em>et al.</em> 2022; Poggiato
            <em>et al.</em> 2022)</span>. Although the two are
            complementary, and answer long-standing calls to include
            interactions within species distribution models <span
            class="citation" data-cites="Wisz2013RolBio">(Wisz <em>et
            al.</em> 2013)</span>, predicting networks in space is a
            different task serving a different goal: focusing first on
            the distribution of network structures and its drivers
            rather than on the distribution of species.</p>
            <p>Explicit spatial predictions of the structure of species
            interaction networks (downscaled metaweb predictions) are
            essential, as they allow comparisons with extant work for
            species-rich communities. Recent approaches to metaweb
            downscaling combined a regional metaweb with species
            distribution maps to generate local assemblages for European
            tetrapods <span class="citation"
            data-cites="Braga2019SpaAna OConnor2020UnvFoo Galiana2021SpaSca Gauzere2022DivBio">(Braga
            <em>et al.</em> 2019; Galiana <em>et al.</em> 2021; Gaüzère
            <em>et al.</em> 2022; O’Connor <em>et al.</em> 2020)</span>,
            Barents Sea marine taxa <span class="citation"
            data-cites="Kortsch2019FooStr">(Kortsch <em>et al.</em>
            2019)</span>, and North Sea demersal fishes and benthic
            epifauna <span class="citation"
            data-cites="Frelat2022FooWeb">(Frelat <em>et al.</em>
            2022)</span>. These downscaled assemblages open up novel
            ways to study network structures, such as assessing changes
            in food web structure across space <span class="citation"
            data-cites="Braga2019SpaAna">(Braga <em>et al.</em>
            2019)</span>, or describing the scaling of Network Area
            Relationships <span class="citation"
            data-cites="Galiana2021SpaSca">(NARs, Galiana <em>et
            al.</em> 2021)</span>. Other examples have shown that the
            metaweb can be used to investigate large-scale variation in
            food web structure, indicating high geographical connections
            and heterogeneous robustness against species extinctions
            <span class="citation" data-cites="Albouy2019MarFis">(Albouy
            <em>et al.</em> 2019)</span>, which are only apparent when
            the local and global networks are <em>both</em> available.
            Further comparisons between network structure and other
            community properties are relevant as they may highlight new
            and surprising elements regarding network biogeography. For
            instance, <span class="citation"
            data-cites="Frelat2022FooWeb">Frelat <em>et al.</em>
            (2022)</span> found a strong spatial coupling between
            community composition and food web structure, but a temporal
            mismatch depending on the spatial scale. <span
            class="citation" data-cites="Poisot2017HosPar">Poisot <em>et
            al.</em> (2017)</span> found that interaction uniqueness
            captures more composition variability than community
            uniqueness, and that sites with exceptional compositions
            might differ for networks and communities, because species
            distributions and species interactions had different
            bioclimatic drivers. Spatialized network data will allow
            these comparisons, identifying important conservation
            targets for networks and whether they differ geographically
            from areas currently prioritized for biodiversity
            conservation.</p>
            <p>Yet, downscaling a regional metaweb towards local network
            predictions that reflect the spatial variability of
            interactions remains an important methodological challenge.
            Even when the metaweb is known, local networks have been
            shown to vary substantially and differ both amongst
            themselves and from the metaweb <span class="citation"
            data-cites="McLeod2021SamAsy">(McLeod <em>et al.</em>
            2021)</span>. This emphasizes the need for methods to
            generate local, downscaled network predictions. A potential
            limitation to previous downscaling approaches is that they
            assumed interactions are equiprobable across space, which
            ignores well-documented interaction variability, and masks
            the effect of environmental conditions on interaction
            realization <span class="citation"
            data-cites="Braga2019SpaAna">(Braga <em>et al.</em>
            2019)</span>. As a consequence, this can over-represent
            interactions locally, but also lead to local predicted
            networks that are more homogeneous than they should. In
            contrast, recent studies argued that seeing interactions as
            probabilistic (rather than binary) events allows us to
            account for their variability in space <span
            class="citation" data-cites="Poisot2016StrPro">(Poisot
            <em>et al.</em> 2016)</span> and that this should also be
            reflected at the metaweb level <span class="citation"
            data-cites="Strydom2023GraEmb">(Strydom <em>et al.</em>
            2023)</span>. <span class="citation"
            data-cites="Gravel2019BriElt">Gravel <em>et al.</em>
            (2019)</span> introduced a probabilistic framework
            describing how the metaweb can generate local realizations
            and showed how it could be used for modelling interaction
            distributions. This approach to downscaling is especially
            relevant when combined with <em>in situ</em> observations of
            interactions and local networks to train interaction models.
            However, such data is rarely available across broad spatial
            extents <span class="citation"
            data-cites="Hortal2015SevSho Poisot2021GloKno Windsor2023UsiEco">(Hortal
            <em>et al.</em> 2015; Poisot <em>et al.</em> 2021; Windsor
            <em>et al.</em> 2023)</span>. Spatially replicated
            interaction data required for such models are especially
            challenging to document with large food web such as European
            tetrapod and Canadian mammal metawebs <span class="citation"
            data-cites="Maiorano2020TetSpe Strydom2022FooWeb">(Maiorano
            <em>et al.</em> 2020; Strydom <em>et al.</em> 2022)</span>,
            where hundreds of species result in tens of thousands of
            species pairs that may potentially interact, over
            continental-scale spatial extents. We currently lack a
            downscaling framework that is both probabilistic and can be
            trained without replicated <em>in situ</em> interaction
            data. But the lack of <em>in situ</em> interaction data
            actually constitutes an interesting opportunity: adopting a
            probabilistic view can allow propagating uncertainty, which
            can play a key role in evaluating the quality (and expected
            variability) of the predictions. Assessing model uncertainty
            would enable us to determine to which degree we should trust
            our predictions and to identify what to do to improve the
            current knowledge.</p>
            <p>Here, we present a workflow to downscale a metaweb in
            space, and illustrate it by spatially reconstructing local
            instances of a probabilistic metaweb of Canadian mammals. We
            do so using a probabilistic approach to both species
            distributions and interactions in a system without spatially
            replicated interaction data. We then explore how the spatial
            structure of the downscaled metaweb varies in space and how
            the uncertainty in predicted interactions can be made
            spatially explicit. We further show that the downscaled
            metaweb can highlight important biodiversity areas and bring
            novel ecological insight compared to traditional community
            measures like species richness or compositional uniqueness.
            We conclude by listing key considerations for the validation
            of such predictions.</p>
            <h1 id="methods">Methods</h1>
            <p>In Fig. 1, we present a conceptual overview of the
            predictive pipeline leading to the downscaled metaweb. Its
            components were grouped as non-spatial and spatial data,
            localized site steps (divided into species, species-pair,
            and network level steps), and the final downscaled,
            spatialized metaweb. Throughout these steps, we highlight
            the importance of presenting the uncertainty of interactions
            and their distribution in space, as well as the variability
            in the structure of reconstructed networks. We argue that
            this requires adopting a probabilistic view of both species
            presence and interactions, and incorporating this variation
            between scales.</p>
            <figure>
            <img src="figures/conceptual-figure.png" id="fig:conceptual"
            alt="Figure 1: Conceptual figure of the proposed workflow used to downscale the probabilistic metaweb in space. The workflow has three components: the data, the localized steps, and the final spatial output. The data are composed of spatial data (with information in every cell) and non-spatial data (constant for all of Canada). The localized steps use these data and are performed separately in every cell, first at a single-species level (using distribution data), then for every species pair (adding interaction data from the metaweb), and finally at the network level by combining the results of all species pairs. The final output of the network-level steps contains a downscaled probabilistic metaweb for every cell across the study extent. Note that in order to mitigate some of the fine-scale grain in the data, we present most outputs at the ecoregion scale, with pixel-scale maps in supplementary material." />
            <figcaption aria-hidden="true">Figure 1: Conceptual figure
            of the proposed workflow used to downscale the probabilistic
            metaweb in space. The workflow has three components: the
            data, the localized steps, and the final spatial output. The
            data are composed of spatial data (with information in every
            cell) and non-spatial data (constant for all of Canada). The
            localized steps use these data and are performed separately
            in every cell, first at a single-species level (using
            distribution data), then for every species pair (adding
            interaction data from the metaweb), and finally at the
            network level by combining the results of all species pairs.
            The final output of the network-level steps contains a
            downscaled probabilistic metaweb for every cell across the
            study extent. Note that in order to mitigate some of the
            fine-scale grain in the data, we present most outputs at the
            ecoregion scale, with pixel-scale maps in supplementary
            material.</figcaption>
            </figure>
            <h2 id="data">Data</h2>
            <h3 id="metaweb">Metaweb</h3>
            <p>We collected probabilistic interaction data from the
            reconstructed metaweb of trophic interactions between
            Canadian mammals from <span class="citation"
            data-cites="Strydom2022FooWeb">Strydom <em>et al.</em>
            (2022)</span>. This metaweb is a-spatial, <em>i.e.</em>, it
            represents interactions between mammals that can occur
            anywhere in Canada. It contains 163 species, and 3280 links
            with a non-zero probability of interaction. The species list
            for the Canadian metaweb was extracted from the
            International Union for the Conservation of Nature (IUCN)
            checklist <span class="citation"
            data-cites="Strydom2022FooWeb">(Strydom <em>et al.</em>
            2022)</span>. The metaweb was reconstructed using graph
            embedding and phylogenetic transfer learning based on the
            metaweb of European terrestrial mammals, which is itself
            based on a comprehensive survey of interactions reported in
            the scientific literature <span class="citation"
            data-cites="Maiorano2020TetSpe">(Maiorano <em>et al.</em>
            2020)</span> – the European metaweb is likely the most
            extensive collective collection of such interactions
            available today. The Canadian metaweb showed a 91% success
            rate in its correct identification of known interactions
            between Canadian mammals recorded in global databases <span
            class="citation" data-cites="Strydom2022FooWeb">(Strydom
            <em>et al.</em> 2022)</span>. This metaweb is probabilistic,
            which has the advantage of reflecting the likelihood of an
            interaction taking place given the phylogenetic and trait
            match between two species; in other words, the probability
            of an interaction in the metaweb describes our confidence in
            the biological feasibility of this interaction. This allows
            incorporating interaction variability between species
            (<em>i.e.</em>, taking into account that two species may not
            always interact whenever or wherever they occur); however,
            other factors beyond trait and phylogenetic matching
            (<em>e.g.</em>, population densities) will also contribute
            to observed interaction frequencies.</p>
            <h3 id="species-occurrences">Species occurrences</h3>
            <p>Downscaling the metaweb involved combining it with
            species occurrence and environmental data. First, we
            extracted species occurrences from the Global Biodiversity
            Information Facility (GBIF; <a
            href="www.gbif.org">www.gbif.org</a>) for the Canadian
            mammals after reconciling species names between the Canadian
            metaweb and GBIF using the GBIF Backbone Taxonomy <span
            class="citation"
            data-cites="GBIFSecretariat2021GbiBac">(GBIF Secretariat
            2021)</span>. This step removed potential duplicates by
            combining species listed in the Canadian metaweb which were
            considered as a single entity by GBIF. We collected
            occurrences for the updated species list (159 species) using
            the GBIF download API on October 21st 2022 <span
            class="citation" data-cites="GBIF.org2022GbiOcc">(GBIF.org
            2022)</span>. We restricted our query to occurrences with
            coordinates between longitudes 175°W to 45°W and latitudes
            10°N to 90°N. This was meant to collect training data
            covering a broader range than our prediction target (Canada
            only) and include observations in similar environments.
            Then, since GBIF observations represent presence-only data
            and most predictive models require absence data, we
            generated the same number of pseudo-absence data as
            occurrences for every species <span class="citation"
            data-cites="Barbet-Massin2012SelPse">(Barbet-Massin <em>et
            al.</em> 2012)</span>. We weighted candidate sites by their
            distance to a known observation (separately for each
            species) using the <code>DistanceToEvent</code> method from
            the <em>Julia</em> package
            <code>SpeciesDistributionToolkit</code>, making it more
            likely to select sites further away from an observation and
            the known species range. This is because our intent was to
            model the potential distribution of species, capturing wider
            responses to the environment, as the downscaled metaweb we
            aimed to produce is potential in nature (see <em>Downscaled
            metaweb</em> section below). We used the Haversine distance
            between observation and candidate pseudo-absence cells when
            drawing pseudo-absences.</p>
            <h3 id="environmental-data">Environmental data</h3>
            <p>We used species distribution models <span
            class="citation" data-cites="Guisan2005PreSpe">(SDMs, Guisan
            &amp; Thuiller 2005)</span> to project Canadian mammal
            habitat suitability across the country, which we treated as
            information on potential distribution. For each species, we
            related occurrences and pseudo-absences with 19 bioclimatic
            variables from CHELSA <span class="citation"
            data-cites="Karger2017CliHig">(Karger <em>et al.</em>
            2017)</span> and 12 consensus land-cover variables from
            EarthEnv <span class="citation"
            data-cites="Tuanmu2014Glo1km">(Tuanmu &amp; Jetz
            2014)</span>. The CHELSA bioclimatic variables
            (<em>BIO1</em>-<em>BIO19</em>) represent various measures of
            temperature and precipitation (<em>e.g.</em>, annual
            averages, monthly maximum or minimum, seasonality) and are
            available for land areas across the globe. We used the most
            recent version, the CHELSA v2.1 dataset <span
            class="citation" data-cites="Karger2021CliHig">(Karger
            <em>et al.</em> 2021)</span>, and subsetted it to land
            surfaces only using the CHELSA v1.2 <span class="citation"
            data-cites="Karger2018DatCli">(Karger <em>et al.</em>
            2018)</span>, which does not cover open water (this is
            appropriate as the data we use as input only cover
            terrestrial mammals). The EarthEnv land-cover variables
            represent classes such as Evergreen broadleaf trees,
            Cultivated and managed vegetation, Urban/Built-up, and Open
            Water. Values range between 0 and 100 and represent the
            consensus prevalence of each class in percentage within a
            pixel (hereafter called a “site”). We coarsened both the
            CHELSA and EarthEnv data from their original 30 arc-second
            resolution to a 2.5 arc-minute one (around 4.5 km at the
            Equator) using GDAL <span class="citation"
            data-cites="GDAL/OGRcontributors2021GdaOgr">(GDAL/OGR
            contributors 2021)</span>. This resolution compromised
            capturing both local variations and broad-scale patterns
            while limiting computation costs to a manageable level as
            memory requirements rapidly increase with spatial
            resolution.</p>
            <h2 id="analyses">Analyses</h2>
            <h3 id="species-distribution-models">Species distribution
            models</h3>
            <p>Our selection criteria for choosing an SDM algorithm was
            to have a method that generated probabilistic results <span
            class="citation" data-cites="Gravel2019BriElt">(similar to
            Gravel <em>et al.</em> 2019)</span>, including both a
            probability of occurrence for a species in a specific site
            and the uncertainty associated with the prediction. These
            were crucial to obtaining a probabilistic version of the
            metaweb as they were used to create spatial variations in
            the localized interaction probabilities (see next section).
            One suitable method for this is Gradient Boosted Trees with
            a Gaussian maximum likelihood estimator from the
            <code>EvoTrees.jl</code> <em>Julia</em> package (<a
            href="https://github.com/Evovest/EvoTrees.jl">https://github.com/Evovest/EvoTrees.jl</a>).
            This method returns a prediction for every site with an
            average value and a standard deviation, which we considered
            as a measure of uncertainty; specifically, the output of a
            Gaussian MLL BRT is the probability distribution of
            observing the positive (<em>i.e.</em>, presence) class. We
            used the mean and standard deviation of the predicted
            outcome to build a Normal distribution for the probability
            of occurrence of a given species at each site (represented
            as probability distributions on Fig. 1). We trained models
            across the extent chosen for occurrences (longitudes 175°W
            to 45°W and latitudes 10°N to 90°N), then predicted species
            distributions only for Canada. We used the 2021 Census
            Boundary Files from Statistics Canada <span class="citation"
            data-cites="StatisticsCanada2022BouFil">(Statistics Canada
            2022)</span> to set the boundaries for our predictions,
            which gave us 970,698 sites in total. Performance evaluation
            for the single species SDMs are available on <a
            href="https://github.com/PoisotLab/SpatialProbabilisticMetaweb/blob/main/data/input/sdm_fit_results.csv">GitHub</a>.</p>
            <h3
            id="building-site-level-instances-of-the-metaweb">Building
            site-level instances of the metaweb</h3>
            <p>The next part of the workflow was to produce local
            metawebs for every site (<em>Localized steps</em> box on
            Fig. 1). This component was divided into single-species,
            two-species, and network-level steps.</p>
            <p>The single-species steps represented four possible ways
            to account for uncertainty in the species distributions and
            bring variation to the spatial metaweb. We explored four
            different options to select a value (<em>P(occurrence)</em>;
            Fig. 1) from the occurrence distributions obtained in the
            previous steps: 1) taking the mean from the distribution as
            the probability of occurrence (option 1 in Fig. 1); 2)
            converting the mean value to a binary one using a specific
            threshold per species (option 2); 3) sampling a random value
            within the Normal distribution (option 3); or 4) converting
            a random value into a binary result (option 4, using a
            separate draw from option 3 and the same threshold as in
            option 2). The threshold (<span
            class="math inline"><em>τ</em></span> in Fig. 1) used was
            the value that maximized Youden’s <em>J</em> informedness
            statistic <span class="citation"
            data-cites="Youden1950IndRat">(Youden 1950)</span>, the same
            metric used by <span class="citation"
            data-cites="Strydom2022FooWeb">Strydom <em>et al.</em>
            (2022)</span> at an intermediate step while building the
            metaweb. The four sampling options were intended to explore
            how uncertainty and variation in the species distributions
            can affect the metaweb result. We expected thresholding to
            have a more pronounced effect on network structure as it
            should reduce the number of links by removing many of the
            rare interactions <span class="citation"
            data-cites="Poisot2016StrPro">(Poisot <em>et al.</em>
            2016)</span>. On the other hand, we expected random sampling
            to create higher spatial heterogeneity compared to the mean
            probabilities, as including some extreme values should
            confound the main trends promoted by environmental
            gradients. We chose option 1 to present our results as it is
            intuitive and essentially represents the result of a
            probabilistic SDM <span class="citation"
            data-cites="Gravel2019BriElt">(as in Gravel <em>et al.</em>
            2019)</span>, but results obtained with other sampling
            strategies are available in Supplementary Material.</p>
            <p>Next, the two-species steps were aimed at assigning a
            probability of observing an interaction between two species
            in a given site. For each species pair, we multiplied the
            product of the two species’ occurrence probabilities
            (<em>P(co-occurrence</em>); Fig. 1) (obtained using one of
            the sampling options above) by their interaction probability
            in the Canadian metaweb. For cases where species in the
            Canadian metaweb were considered as the same species by the
            GBIF Backbone Taxonomy (the reconciliation step mentioned
            earlier), we used the highest interaction probabilities
            involving the duplicated species.</p>
            <p>The network-level steps then created the probabilistic
            metaweb for the site. We assembled all the local interaction
            probabilities (from the two-species steps) into a
            probabilistic network <span class="citation"
            data-cites="Poisot2016StrPro">(Poisot <em>et al.</em>
            2016)</span>. We then sampled several random network
            realizations to represent the potential local realization
            process <span class="citation"
            data-cites="Poisot2015SpeWhy">(Poisot <em>et al.</em>
            2015)</span>. This resulted in a distribution of localized
            networks, which we averaged over the number of simulations
            to obtain a single probabilistic network for the site.</p>
            <h3 id="downscaled-metaweb">Downscaled metaweb</h3>
            <p>The final output of our workflow was the downscaled
            metaweb, which contains a localized probabilistic metaweb in
            every site across the study area (<em>Outputs</em> box in
            Fig. 1). The metaweb sets an upper bound on the potential
            interactions <span class="citation"
            data-cites="Strydom2023GraEmb">(Strydom <em>et al.</em>
            2023)</span>, therefore, the downscaled metaweb is a refined
            upper boundary at the local scale that takes into account
            co-occurrences. It is still potential in nature and differs
            from a local realization, from which it should have a
            different structure. Nonetheless, from the downscaled
            metaweb, we can create maps of network properties
            (<em>e.g.</em>, number of links, connectance) measured on
            the local probabilities of species interactions and
            occurrences, and compute some traditional community-level
            measures such as species richness. We chose to compute and
            display the expected number of links <span class="citation"
            data-cites="Poisot2016StrPro Gravel2019BriElt">(measured on
            probabilistic networks following Poisot <em>et al.</em>
            2016; see Gravel <em>et al.</em> 2019 for a similar
            example)</span> as its relationship with species richness
            has been highlighted in a spatial context in recent studies
            <span class="citation"
            data-cites="Galiana2021SpaSca Galiana2022EcoNet">(Galiana
            <em>et al.</em> 2021, 2022)</span>. We also computed the
            uncertainty associated with the community and network
            measurements (richness variance and link variance,
            respectively) and compared their spatial distribution (see
            Supplementary Material).</p>
            <h3 id="analyses-of-results-by-ecoregions">Analyses of
            results by ecoregions</h3>
            <p>Since both species composition and network summary values
            display a high spatial variation and complex patterns, we
            simplified the representation of their distribution by
            grouping sites by ecoregion, as species and interaction
            composition have been shown to differ between ecoregions
            across large spatial scales <span class="citation"
            data-cites="Martins2022GloReg">(Martins <em>et al.</em>
            2022)</span>. To do so, we rasterized the Canadian subset of
            the global map of ecoregions from <span class="citation"
            data-cites="Dinerstein2017EcoApp Martins2022GloReg">(Dinerstein
            <em>et al.</em> 2017; also used by Martins <em>et al.</em>
            2022)</span>, which resulted in 44 different ecoregions. For
            every measure we report (<em>e.g.</em>, species richness,
            number of links), we calculated the median site value for
            each ecoregion, as a way to avoid bias due to long tails in
            the distributions. We also measured within-ecoregion
            variation as the 89% interquantile range of the site values
            in each ecoregion <span class="citation"
            data-cites="McElreath2020StaRet">(threshold chosen to avoid
            confusion with conventional significance tests; McElreath
            2020)</span>.</p>
            <h3 id="analyses-of-ecological-uniqueness">Analyses of
            ecological uniqueness</h3>
            <p>We compared the compositional uniqueness of the networks
            and the communities to assess whether they indicated areas
            of exceptional composition. We measured uniqueness using the
            local contributions to beta diversity <span class="citation"
            data-cites="Legendre2013BetDiv">(LCBD, Legendre &amp; De
            Cáceres 2013)</span>, which identify sites with exceptional
            composition by quantifying how much one site contributes to
            the total variance in the community composition. While many
            studies used LCBD values to evaluate uniqueness on local
            scales or few study sites <span class="citation"
            data-cites="daSilva2014LocReg Heino2017ExpSpe">(for example,
            da Silva &amp; Hernández 2014; Heino &amp; Grönroos
            2017)</span>, recent studies used the measure on predicted
            species compositions over broad spatial extents and a large
            number of sites <span class="citation"
            data-cites="Vasconcelos2018ExpImp Dansereau2022EvaEco">(Dansereau
            <em>et al.</em> 2022; Vasconcelos <em>et al.</em>
            2018)</span>. LCBD values can also be used to measure
            uniqueness for networks by computing the values over the
            adjacency matrix, which has been shown to capture more
            unique sites and uniqueness variability than through species
            composition <span class="citation"
            data-cites="Poisot2017HosPar">(Poisot <em>et al.</em>
            2017)</span>. Here, we measured and compared the uniqueness
            of our localized community and network predictions. For
            species composition, we assembled a site-by-species
            community matrix (970,698 sites by 159 species) with the
            probability of occurrence of each species at every site
            obtained in the species distribution models. For network
            composition, we assembled a site-by-interaction matrix with
            the localized probability of interaction at every site given
            by the downscaled metaweb (therefore, 970,698 sites by 3,108
            interactions with defined probabilities in the metaweb). We
            applied the Hellinger transformation on both matrices and
            computed the LCBD values from the total variance in the
            matrices <span class="citation"
            data-cites="Legendre2013BetDiv">(Legendre &amp; De Cáceres
            2013)</span>. High LCBD values indicate a high contribution
            to the overall variance and a unique species or interaction
            composition compared to other sites. Since the values
            themselves are very low given our high number of sites <span
            class="citation" data-cites="Dansereau2022EvaEco">(as in
            Dansereau <em>et al.</em> 2022)</span>, what matters
            primarily is the magnitude of the difference between the
            sites. Given this, we divided values by the maximum value in
            each matrix (species or network) and suggest that these
            should be viewed as relative contributions compared to the
            highest observed contribution. As with other measures, we
            then summarized the local uniqueness values by ecoregion by
            taking the median LCBD value and measuring the 89%
            interquantile range.</p>
            <h3 id="analyses-of-network-motifs">Analyses of network
            motifs</h3>
            <p>To further explore network structure in space, we
            investigated the distribution of network motifs across
            space. Motifs are defined sets of interaction between
            species <span class="citation"
            data-cites="Milo2002NetMot Stouffer2007EviExi">(Milo <em>et
            al.</em> 2002; Stouffer <em>et al.</em> 2007)</span>, for
            instance two predators sharing one prey, which are repeated
            within larger and more complex food webs. Motifs are linked
            to community persistence <span class="citation"
            data-cites="Stouffer2010UndFoo">(Stouffer &amp; Bascompte
            2010)</span> and community structure <span class="citation"
            data-cites="Cirtwill2015ConPre Simmons2019MotBip">(Cirtwill
            &amp; Stouffer 2015; Simmons <em>et al.</em> 2019)</span>,
            are conserved across scales <span class="citation"
            data-cites="Baker2015SpeRol Baiser2016MotAss">(Baiser <em>et
            al.</em> 2016; Baker <em>et al.</em> 2015)</span>, and are
            part of a common backbone of interactions among all food
            webs <span class="citation"
            data-cites="Mora2018IdeCom">(Mora <em>et al.</em>
            2018)</span>. We focused on four of the most studied
            three-species motifs <span class="citation"
            data-cites="Stouffer2007EviExi Stouffer2010UndFoo Baiser2016MotAss">(Baiser
            <em>et al.</em> 2016; Stouffer <em>et al.</em> 2007;
            Stouffer &amp; Bascompte 2010)</span>: S1 (tri-trophic food
            chains), S2 (omnivory), S4 (exploitative competition) and S5
            (apparent competition). These motifs can be grouped into two
            pairs according to their ecological information: S1 and S2
            highlight different trophic structures, while S4 and S5
            indicate different competition types. Therefore, we compared
            the spatial distribution of the motifs in each pair to see
            which ones were dominant across all our sites. First, we
            computed the expected motif count for each of the four
            motifs for all sites using the localized probabilistic
            networks from the downscaled metaweb <span class="citation"
            data-cites="Poisot2016StrPro">(following Poisot <em>et
            al.</em> 2016)</span>. Then, we compared the expected counts
            of the motifs within the two pairs. To do so, we used a
            normalized difference measure similar to the Normalized
            Difference Vegetation Index (NDVI), where we compute the
            difference between the two motif counts over their sum. We
            called the index comparing the two trophic motifs (S1 and
            S2) the Normalized Difference Trophic Index (NDTI) and the
            one comparing the competition motifs (S4 and S5) the
            Normalized Difference Competition Index. We defined both
            indexes as: <span class="math display">$$NDTI =
            \frac{(S1 - S2)}{(S1 +S2)}$$</span> <span
            class="math display">$$NDCI = \frac{(S4 - S5)}{(S4
            +S5)}$$</span> Values for both indexes are bounded between
            -1 and 1. A value of 0 indicates that both motifs have the
            same expected counts. Positive values indicate that the
            first motif in each index (S1 and S4) is dominant and has a
            higher expected count, while negative values indicate that
            the second motif (S2 and S5) is dominant. As with previous
            measures, we then summarized both index values by ecoregion
            by taking the ecoregion median and measuring its
            within-ecoregion variation with the 89% interquantile range.
            Ecoregion values therefore indicate if one type of trophic
            structure (for NDTI) and one type of competition (for NDCI)
            is dominant in the ecoregion, while the interquantile range
            values measure whether the dominant type varies within the
            ecoregion.</p>
            <p>We used <em>Julia</em> v1.9.0 <span class="citation"
            data-cites="Bezanson2017JulFre">(Bezanson <em>et al.</em>
            2017)</span> to implement all our analyses. We used packages
            <code>GBIF.jl</code> <span class="citation"
            data-cites="Dansereau2021SimJl">(Dansereau &amp; Poisot
            2021)</span> to reconcile species names using the GBIF
            Backbone Taxonomy,
            <code>SpeciesDistributionToolkit.jl</code> (<a
            href="https://github.com/PoisotLab/SpeciesDistributionToolkit.jl">https://github.com/PoisotLab/SpeciesDistributionToolkit.jl</a>)
            to handle raster layers, species occurrences and generate
            pseudo-absences (using the <code>DistanceToEvent</code>
            method), <code>EvoTrees.jl</code> (<a
            href="https://github.com/Evovest/EvoTrees.jl">https://github.com/Evovest/EvoTrees.jl</a>)
            to perform the Gradient Boosted Trees,
            <code>EcologicalNetworks.jl</code> <span class="citation"
            data-cites="Poisot2019EcoJl">(Poisot <em>et al.</em>
            2019)</span> to analyze network and metaweb structure, and
            <code>Makie.jl</code> <span class="citation"
            data-cites="Danisch2021MakJl">(Danisch &amp; Krumbiegel
            2021)</span> to produce figures. Our data sources (CHELSA,
            EarthEnv, Ecoregions) were all unprojected, and we did not
            use a projection in our analyses. However, we displayed the
            results using a Lambert conformal conic projection more
            appropriate for Canada using <code>GeoMakie.jl</code> (<a
            href="https://github.com/MakieOrg/GeoMakie.jl">https://github.com/MakieOrg/GeoMakie.jl</a>).
            All the code used to implement our analyses is archived on
            Zenodo <span class="citation"
            data-cites="Dansereau2023PoiSpa">(<a
            href="https://doi.org/10.5281/zenodo.8350065" class="uri"
            role="doc-biblioref">https://doi.org/10.5281/zenodo.8350065</a>;
            Dansereau &amp; Poisot 2023)</span> and includes
            instructions on how to run a smaller example at a coarser
            resolution. Note that running our analyses at full scale is
            resource and memory-intensive and required the use of
            computer clusters provided by Calcul Québec and the Digital
            Research Alliance of Canada. Full-scale computations
            (excluding motifs) required 900 CPU core-hours and peaked at
            500 GB of RAM. Computations for network motifs were
            susbtantially more demanding, requiring about 12 CPU
            core-years (approx. <span
            class="math inline">10<sup>5</sup></span> hours).</p>
            <h1 id="results">Results</h1>
            <p>Our workflow allowed us to display the spatial
            distribution of ecoregion-level community measures (here,
            expected species richness) and network measures (expected
            number of links; Fig. 2). We highlight that the measures
            presented here are first computed over the predicted
            communities and networks obtained when downscaling the
            metaweb, then summarized across the ecoregions (taking the
            median within each ecoregion). They are not a direct
            prediction of the measure itself (<em>e.g.</em>, we do not
            present a prediction of actual species richness at each
            location). Expected ecoregion richness (Fig. 2A) and
            expected number of links (Fig. 2B) displayed similar
            distributions with a latitudinal gradient and higher values
            in the south. Within-ecoregion variability was distributed
            slightly differently with a less constant latitudinal
            gradient, notably lower interquantile ranges near the
            southern border (for example, near Vancouver Island and the
            Rockies on the West Coast, and near the Ontario Peninsula,
            the Saint-Lawrence Valley, and Central New-Brunswick in the
            East; Fig. 2C-D). Bivariate comparison of the distributions
            of species richness and expected number of links and of
            their respective within-ecoregion variability further shows
            some areas of mismatches, indicating that richness and links
            do not co-vary completely although they may show similar
            distributions for median values (see Supplementary Material,
            Fig. S1). All results shown are based on the first sampling
            strategy (option 1) mentioned in the <em>Building site-level
            instances of the metaweb</em> section, where we used the
            mean value of the species distributions as the species
            occurrence probabilities (results for other sampling
            strategies are shown in Supplementary Material, Fig. S2).
            Site-level results (before summarizing by ecoregion) are
            also provided in Supplementary Material (Figs. S3-S6).</p>
            <figure>
            <img src="figures/ecoregion_comparison_iqr.png"
            id="fig:ecoregion_measures"
            alt="Figure 2: (A-B) Example of a community measure (A, expected species richness) and a network one (B, expected number of links). Both measures are assembled from the predicted probabilistic communities and networks, respectively. Values are first measured separately for all sites; then, the median value within each ecoregion was taken to represent the ecoregion-level value. (C-B) Representation of the 89% interquantile range of values within the ecoregion for expected richness (C) and expected number of links (D). All colour bars follow a log scale with tick marks representing even intervals. Real values (non-log transformed) are shown beside major tick marks while minor ticks represent half increments." />
            <figcaption aria-hidden="true">Figure 2: (A-B) Example of a
            community measure (A, expected species richness) and a
            network one (B, expected number of links). Both measures are
            assembled from the predicted probabilistic communities and
            networks, respectively. Values are first measured separately
            for all sites; then, the median value within each ecoregion
            was taken to represent the ecoregion-level value. (C-B)
            Representation of the 89% interquantile range of values
            within the ecoregion for expected richness (C) and expected
            number of links (D). All colour bars follow a log scale with
            tick marks representing even intervals. Real values (non-log
            transformed) are shown beside major tick marks while minor
            ticks represent half increments.</figcaption>
            </figure>
            <p>Our results also indicate a mismatch between the
            uniqueness of communities and networks (Fig. 3). Uniqueness
            was higher mostly in the north and along the south border
            for communities, but mainly in the north for networks
            (Fig. 3A-B). Consequently, ecoregions with both unique
            community composition and unique network composition were
            mostly in the north (Fig. 3C). Meanwhile, some areas were
            unique for one element but not the other. For instance,
            ecoregions along the south border had a unique species
            composition but a more common network composition (Fig. 3C).
            Two ecoregions showed the opposite (unique network
            compositions only) at higher latitudes (Davis Highlands
            tundra, near 70°N, and Southern Hudson Bay taiga, near
            54°N). Moreover, network uniqueness values for ecoregions
            spanned a narrower range between the 44 ecoregions than
            species LCBD values (Fig. 3D, left). Within-ecoregion
            variation was also lower for network values with generally
            lower 89% interquantile ranges among the site-level LCBD
            values (Fig. 3D, right).</p>
            <figure>
            <img src="figures/ecoregion_LCBD_4panels.png"
            id="fig:ecoregion_lcbd"
            alt="Figure 3: (A-B) Representation of the ecoregion uniqueness values based on species composition (A) and network composition (B). LCBD values were first computed across all sites and scaled relative to the maximum value observed. The ecoregion LCBD value is the median value for the sites in the ecoregion. (C) Bivariate representation of species and network composition LCBD. Values are grouped into three quantiles separately for each variable. The colour combinations represent the nine possible combinations of quantiles. The species uniqueness (horizontal axis) goes left to right from low uniqueness (light grey, bottom left) to high uniqueness (green, bottom right). The network composition uniqueness goes bottom-up from low uniqueness (light grey, bottom left) to high uniqueness (blue, top left). (D) Probability densities for the ecoregion LCBD values for species and network LCBD (left), highlighting the variability of LCBD values between ecoregions, and the 89% interquartile range of the values within each ecoregion (right), highlighting the variability within the ecoregions." />
            <figcaption aria-hidden="true">Figure 3: (A-B)
            Representation of the ecoregion uniqueness values based on
            species composition (A) and network composition (B). LCBD
            values were first computed across all sites and scaled
            relative to the maximum value observed. The ecoregion LCBD
            value is the median value for the sites in the ecoregion.
            (C) Bivariate representation of species and network
            composition LCBD. Values are grouped into three quantiles
            separately for each variable. The colour combinations
            represent the nine possible combinations of quantiles. The
            species uniqueness (horizontal axis) goes left to right from
            low uniqueness (light grey, bottom left) to high uniqueness
            (green, bottom right). The network composition uniqueness
            goes bottom-up from low uniqueness (light grey, bottom left)
            to high uniqueness (blue, top left). (D) Probability
            densities for the ecoregion LCBD values for species and
            network LCBD (left), highlighting the variability of LCBD
            values between ecoregions, and the 89% interquartile range
            of the values within each ecoregion (right), highlighting
            the variability within the ecoregions.</figcaption>
            </figure>
            <p>Comparing the distribution of dominant network motifs
            revealed additional areas of variation in network structure
            (Fig. 4). NDTI displayed a latitudinal gradient between the
            trophic motifs. Northern ecoregions showed positive NDTI
            values and high dominance of S1 (tri-trophic chains)
            expected counts compared to S2 (omnivory) but ecoregions
            along the south border showed a reduced dominance (Fig. 4A).
            Ecoregions near the Ontario Peninsula and Saint-Lawrence
            Valley showed values close to zero, indicating a balance
            between two motifs, while Central New Brunswick had slightly
            negative values, indicating a low dominance of S2. In
            comparison, NDCI values showed an evenly high dominance of
            S5 (apparent competition) over S4 (exploitative competition)
            across all ecoregions (Fig. 4B). Meanwhile, within-ecoregion
            variance displayed a different spatial distribution from the
            median values. NDTI interquantile ranges spanned a wide
            range of values and were higher both in the north and in the
            south (although not in the ecoregions with higher NDTI
            median values) (Fig. 4C). On the other hand, NDCI
            interquantile ranges showed lower within-ecoregion variance
            in most ecoregions except in the northernmost one (Canadian
            High Arctic tundra), which has a notably higher value­
            (Fig. 4D). Although this higher variance does not reflect in
            the NDCI median values, it does appear when looking at the
            site-level values, where this ecoregion is the only one with
            patches with high positive NDCI values (indicating a
            dominance of S4) surrounded by highly negative values
            (indicating a dominance of S5) as in other ecoregions (Fig.
            S6B).</p>
            <figure>
            <img src="figures/motifs_ecoregion_NDI_4panels.png"
            id="fig:motifs"
            alt="Figure 4: Comparison of the dominant ecological motifs across ecoregions. A) Normalized Difference Index (NDTI) comparing the trophic motifs S1 (tri-trophic food chains) and S2 (omnivory). Positive values indicate a dominance of S1, while negative values indicate a dominance of S2. Values equal or superior to |0.5| are shown with the same colour as they indicate a high dominance of one motif. B) Normalized Difference Index (NDCI) comparing the competition motifs S4 (exploitative competition) and S5 (apparent competition). Positive values indicate a dominance of S4, while negative values indicate a dominance of S5. (C-D) Representation of the 89% interquantile range of values within the ecoregion for the trophic motifs index (NDTI, C) and competition motifs index (NDCI, D)." />
            <figcaption aria-hidden="true">Figure 4: Comparison of the
            dominant ecological motifs across ecoregions. A) Normalized
            Difference Index (NDTI) comparing the trophic motifs S1
            (tri-trophic food chains) and S2 (omnivory). Positive values
            indicate a dominance of S1, while negative values indicate a
            dominance of S2. Values equal or superior to |0.5| are shown
            with the same colour as they indicate a high dominance of
            one motif. B) Normalized Difference Index (NDCI) comparing
            the competition motifs S4 (exploitative competition) and S5
            (apparent competition). Positive values indicate a dominance
            of S4, while negative values indicate a dominance of S5.
            (C-D) Representation of the 89% interquantile range of
            values within the ecoregion for the trophic motifs index
            (NDTI, C) and competition motifs index (NDCI,
            D).</figcaption>
            </figure>
            <h1 id="discussion">Discussion</h1>
            <p>Our approach presents a way to downscale a metaweb,
            produce localized predictions using probabilistic networks
            as inputs and outputs, and incorporate uncertainty, as
            called for by <span class="citation"
            data-cites="Strydom2023GraEmb">Strydom <em>et al.</em>
            (2023)</span>. It gives us an idea of what local metawebs or
            networks could look like in space, given species
            distributions and their variability, as well as the
            uncertainty around species interactions. We also provide the
            first spatial representation of the metaweb of Canadian
            mammals <span class="citation"
            data-cites="Strydom2022FooWeb">(Strydom <em>et al.</em>
            2022)</span> and a probabilistic equivalent to how the
            European tetrapod metaweb <span class="citation"
            data-cites="Maiorano2020TetSpe">(Maiorano <em>et al.</em>
            2020)</span> was used to predict localized networks in
            Europe <span class="citation"
            data-cites="Braga2019SpaAna OConnor2020UnvFoo Galiana2021SpaSca Gauzere2022DivBio Botella2023LanInt">(Botella
            <em>et al.</em> 2023; Braga <em>et al.</em> 2019; Galiana
            <em>et al.</em> 2021; Gaüzère <em>et al.</em> 2022; O’Connor
            <em>et al.</em> 2020)</span>. Therefore, our approach could
            open similar possibilities of investigations on the
            variation of structure in space <span class="citation"
            data-cites="Braga2019SpaAna">(Braga <em>et al.</em>
            2019)</span> and on the effect of land-use intensification
            <span class="citation"
            data-cites="Botella2023LanInt">(Botella <em>et al.</em>
            2023)</span> on North American food webs, particularly
            Canadian mammal food webs. Other interesting research
            applications include assessing climate change impacts on
            network structure <span class="citation"
            data-cites="Kortsch2015CliCha">(<em>e.g.</em>, Kortsch
            <em>et al.</em> 2015)</span> or investigating linkages
            between network structure and stability <span
            class="citation" data-cites="Windsor2023UsiEco">(Windsor
            <em>et al.</em> 2023)</span>.</p>
            <p>As our approach is probabilistic, it does not assume
            species interact whenever they co-occur and incorporates
            variability based on environmental conditions (via projected
            species distributions), which could lead to different
            results by introducing a different association between
            species richness and network properties. <span
            class="citation" data-cites="Galiana2021SpaSca">Galiana
            <em>et al.</em> (2021)</span> found that species richness
            had a large explanatory power over network properties, but
            mentioned this could potentially be due to interactions
            between species being constant across space. Here, we found
            that potential species richness and number of links
            displayed similar distributions following a latitudinal
            gradient, but that the within-ecoregion variance was lower
            along the southern border than the measures themselves
            (Fig. 2). The causes for this lack of consistency at the
            eco-region scale could be verified in future studies; for
            instance, higher urban density in the south can create more
            heterogeneity within an eco-region, and the variability in
            network structure may reflect true landscape variability
            within eco-regions. We found that network density (links on
            Fig. 2B) was lower in the north, which is contrary to what
            was observed in Europe for the terrestrial tetrapod metaweb
            <span class="citation"
            data-cites="Braga2019SpaAna Galiana2021SpaSca">(Braga <em>et
            al.</em> 2019; Galiana <em>et al.</em> 2021)</span> and for
            willow-galler-parasitoid networks <span class="citation"
            data-cites="Gravel2019BriElt">(Gravel <em>et al.</em>
            2019)</span>, where connectance was higher in northern
            regions. However, those are systems with different numbers
            of species and environmental conditions (<em>e.g.</em>,
            Europe and Canada could differ due to varying climatic
            conditions, land-use, and species composition at the same
            latitudes). Further research should investigate why these
            results might differ between continents and ecological
            systems and whether it is due to the methodology, data, or
            biogeographical processes.</p>
            <p>Our LCBD and uniqueness results highlighted that areas
            with unique network composition differ from sites with
            unique species composition. In other words, the joint
            distribution of community and network uniqueness highlights
            different diversity hotspots. <span class="citation"
            data-cites="Poisot2017HosPar">Poisot <em>et al.</em>
            (2017)</span> showed a similar result with host-parasite
            communities of rodents and ectoparasitic fleas. Our results
            further show that these differences could be distributed
            across ecoregions and over a broad spatial extent for mammal
            food webs. LCBD scores essentially highlight variability
            hotspots and are a measure of the variance of community or
            network structure. Here, they also serve as an
            inter-ecoregion variation measure, which can be compared to
            the within-ecoregion variation highlighted by the
            interquantile ranges. The narrower range of values for
            network LCBD values and the lower IQR values indicate that
            both the inter-ecoregion and within-ecoregion variation are
            lower for networks than for species (Fig. 3).</p>
            <p>Our analysis of the distribution of dominant network
            motifs revealed additional areas of variation in network
            structure. Trophic motifs (S1 and S2, measured through NDTI
            values) showed a latitudinal gradient different from the
            richness and links ones, with high dominance of tri-trophic
            chains (S1) in the north and higher omnivory counts (S2)
            only in a few ecoregions in the south. These results did not
            seem related to ecoregion variance, which once again showed
            a very different distribution from the median values.
            Meanwhile, competition motifs (S4 and S5, measured through
            NDCI values) showed an even dominance of apparent
            competition (S5) but high variance in a single ecoregion.
            Overall, our results show that dominant motifs within a
            mammal food web not only vary between ecoregions, but
            actually vary differently across space.</p>
            <p>When to use the workflow we presented here will depend on
            the availability of interaction data or existing metawebs,
            and on the intent to incorporate interaction variability, as
            well as ecoregion-level variability. In systems where <em>in
            situ</em> interaction and complete network data are
            available, the approach put forward by <span
            class="citation" data-cites="Gravel2019BriElt">Gravel <em>et
            al.</em> (2019)</span> achieves a similar purpose as we
            attempted here, but is more rigorous and allows modelling
            the effect of the environment on the interactions
            themselves. Without such data, establishing or predicting
            the metaweb (<em>e.g.</em>, using transfer learning) should
            be the first step toward producing localized predictions
            <span class="citation"
            data-cites="Strydom2023GraEmb">(Strydom <em>et al.</em>
            2023)</span>. Our framework then downscales the metaweb
            towards the localized predictions, here using the
            probabilistic Canadian mammal one, but it can also use other
            metawebs generated through various means. Well-documented
            binary ones such as the European tetrapod metaweb could be
            partly combined with our approach if used with probabilistic
            SDMs and summarized by ecoregions (as they would only lack
            an initial probabilistic metaweb, but would still obtain a
            more probabilistic output). Our approach will essentially
            differ from previous attempts in how it perceives
            uncertainty and variability. For instance, rare interactions
            should not be over-represented <span class="citation"
            data-cites="Poisot2016StrPro">(Poisot <em>et al.</em>
            2016)</span> and should have lesser effects over computed
            network measures. Furthermore, summarizing results by
            ecoregion allows for showing variation within and between
            ecologically meaningful biogeographic boundaries <span
            class="citation" data-cites="Martins2022GloReg">(Martins
            <em>et al.</em> 2022)</span>, which, as our results showed,
            is not constant across space and can help identify
            contrasting diversity hotspots.</p>
            <p>Although our approach can generate a wealth of
            predictions, the next step is quite obviously to work on the
            validation of these predictions. This step is mandatory if
            the predictions are to be made actionable. Nevertheless,
            developing a way to generate these predictions when
            information is initially scarce, as we present here, is
            highly important in itself: it establishes a baseline for
            what the expected measurements will look like, but also
            (through quantification of variability and uncertainty)
            provides an estimate of where a more sustained effort is
            required to adequately document food web structure. For
            instance, documenting interactions at a single location in
            an eco-region with high variability of predicted network
            structure may ultimately be less informative, whereas more
            homogeneous eco-regions constitute “low-hanging fruits” for
            validation. Any future sampling of food web structure can be
            used to (in)validate the predictions: they can be fed into
            the model to iterate these results again, and by decreasing
            the uncertainty associated to the interactions, can boost
            the accuracy of the entire model. There are two external
            sources of information that can also increase the predictive
            ability of the model. Because the downscaling relies on
            SDMs, any additional documentation of species presences can
            be reflected in the network prediction. Furthemore, because
            the metaweb itself was obtained through transfer learning
            from data describing a different system, any change to the
            knowledge in this other data can be reflected in the input
            data. As <span class="citation"
            data-cites="Strydom2023GraEmb">Strydom <em>et al.</em>
            (2023)</span> point out, validation of metaweb predictions,
            empirical sampling, and method design should all proceed
            jointly, and making conceptual progress in one of these
            areas helps all the others. In this manuscript, we focused
            on motif composition, both for its relevance to functional
            properties of food webs, but also because it can be fairly
            reliably infered from partial network data; in other words,
            validating some predictions is not necessarily relying upon
            exhaustive documentation of local food webs.</p>
            <p>The recent shift in focus towards building metawebs opens
            many opportunities for projections of networks in space
            through probabilistic downscaling, as we suggested here.
            Metawebs have been documented in many systems, allowing us
            to build new ones from predictions. How the European
            tetrapod metaweb <span class="citation"
            data-cites="Maiorano2020TetSpe">(Maiorano <em>et al.</em>
            2020)</span> was used to predict the Canadian mammal metaweb
            <span class="citation"
            data-cites="Strydom2022FooWeb">(Strydom <em>et al.</em>
            2022)</span> is one such case, but recent examples also
            extend to other systems. Metawebs have been compiled for
            many marine food webs <span class="citation"
            data-cites="Kortsch2019FooStr Lopez-Lopez2022EcoNet Kortsch2021DisTem">(<em>e.g.</em>,
            Barents Sea, Kortsch <em>et al.</em> 2019; North Scotia Sea,
            López-López <em>et al.</em> 2022; Gulf of Riga, Kortsch
            <em>et al.</em> 2021)</span> and used to predict the
            probability of novel interactions <span class="citation"
            data-cites="Pecuchet2020NovFee">(Artic food web of the
            Barents sea, Pecuchet <em>et al.</em> 2020)</span>. <span
            class="citation" data-cites="Olivier2019ExpTem">Olivier
            <em>et al.</em> (2019)</span> built a temporally resolved
            metaweb of demersal fish and benthic epifauna but also
            suggested combining their approach with techniques
            estimating the probability of occurrence of trophic
            relationships to describe spatial and temporal variability
            more accurately. <span class="citation"
            data-cites="Lurgi2020GeoVar">Lurgi <em>et al.</em>
            (2020)</span> built a metaweb and probabilistic
            (occurrence-based) networks for rocky intertidal communities
            (and in doing so, they also showed that environmental
            factors do not affect the structure of binary and
            probabilistic networks in different ways). <span
            class="citation" data-cites="Albouy2019MarFis">Albouy <em>et
            al.</em> (2019)</span> predicted the global marine fish food
            web using a probabilistic model, showing the potential to
            describe networks across broad spatial scales. Similarly,
            predictive approaches are also increasingly used with other
            interaction types to highlight interaction hotspots on
            global scales <span class="citation"
            data-cites="Becker2022OptPre Poisot2023NetEmb">(<em>e.g.</em>,
            mapping geographical hotspots of predicted host-virus
            interactions between bats and betacoronaviruses, Becker
            <em>et al.</em> 2022; predicting the distribution of hidden
            interactions in the mammalian virome, Poisot <em>et al.</em>
            2023)</span>. Our workflow offers the potential to bring
            these global predictions down to the local scale where they
            can be made more actionable, and vastly increases the
            diversity of ecological networks that can be projected in
            space.</p>
            <h1 id="acknowledgements">Acknowledgements</h1>
            <p>We acknowledge that this study was conducted on land
            within the traditional unceded territory of the Saint
            Lawrence Iroquoian, Anishinabewaki, Mohawk, Huron-Wendat,
            and Omàmiwininiwak nations. GD is funded by the NSERC
            Postgraduate Scholarship – Doctoral (grant ES D – 558643),
            the FRQNT doctoral scholarship (grant no. 301750), and the
            NSERC CREATE BIOS² program. TP is funded by the Wellcome
            Trust (223764/Z/21/Z), NSERC through the Discovery Grant and
            Discovery Accelerator Supplements programs, and the Courtois
            Foundation. This research was enabled in part by support
            provided by Calcul Québec (<a
            href="http://www.calculquebec.ca">calculquebec.ca</a>) and
            the Digital Research Alliance of Canada (<a
            href="https://alliancecan.ca/">alliance​can​.ca</a>) through
            the Narval general purpose cluster.</p>
            <h1 class="unnumbered" id="references">References</h1>
            <div id="refs"
            class="references csl-bib-body hanging-indent"
            role="doc-bibliography">
            <div id="ref-Albouy2019MarFis" class="csl-entry"
            role="doc-biblioentry">
            Albouy, C., Archambault, P., Appeltans, W., Araújo, M.B.,
            Beauchesne, D., Cazelles, K., <em>et al.</em> (2019). <a
            href="https://doi.org/10.1038/s41559-019-0950-y">The marine
            fish food web is globally connected</a>. <em>Nature Ecology
            &amp; Evolution</em>, 3, 1153–1161.
            </div>
            <div id="ref-Baiser2016MotAss" class="csl-entry"
            role="doc-biblioentry">
            Baiser, B., Elhesha, R. &amp; Kahveci, T. (2016). <a
            href="https://doi.org/10.1111/oik.02532">Motifs in the
            assembly of food web networks</a>. <em>Oikos</em>, 125,
            480–491.
            </div>
            <div id="ref-Baiser2019EcoRul" class="csl-entry"
            role="doc-biblioentry">
            Baiser, B., Gravel, D., Cirtwill, A.R., Dunne, J.A.,
            Fahimipour, A.K., Gilarranz, L.J., <em>et al.</em> (2019).
            <a href="https://doi.org/10.1111/geb.12925">Ecogeographical
            rules and the macroecology of food webs</a>. <em>Global
            Ecology and Biogeography</em>, geb.12925.
            </div>
            <div id="ref-Baker2015SpeRol" class="csl-entry"
            role="doc-biblioentry">
            Baker, N.J., Kaartinen, R., Roslin, T. &amp; Stouffer, D.B.
            (2015). <a
            href="https://doi.org/10.1111/ecog.00913">Species’ roles in
            food webs show fidelity across a highly variable oak
            forest</a>. <em>Ecography</em>, 38, 130–139.
            </div>
            <div id="ref-Barbet-Massin2012SelPse" class="csl-entry"
            role="doc-biblioentry">
            Barbet-Massin, M., Jiguet, F., Albert, C.H. &amp; Thuiller,
            W. (2012). <a
            href="https://doi.org/10.1111/j.2041-210X.2011.00172.x">Selecting
            pseudo-absences for species distribution models: How, where
            and how many?</a> <em>Methods in Ecology and Evolution</em>,
            3, 327–338.
            </div>
            <div id="ref-Becker2022OptPre" class="csl-entry"
            role="doc-biblioentry">
            Becker, D.J., Albery, G.F., Sjodin, A.R., Poisot, T.,
            Bergner, L.M., Chen, B., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1016/S2666-5247(21)00245-7">Optimising
            predictive models to prioritise viral discovery in zoonotic
            reservoirs</a>. <em>The Lancet Microbe</em>, 3, e625–e637.
            </div>
            <div id="ref-Bezanson2017JulFre" class="csl-entry"
            role="doc-biblioentry">
            Bezanson, J., Edelman, A., Karpinski, S. &amp; Shah, V.B.
            (2017). <a href="https://doi.org/10.1137/141000671">Julia: A
            fresh approach to numerical computing</a>. <em>SIAM
            Review</em>, 59, 65–98.
            </div>
            <div id="ref-Botella2023LanInt" class="csl-entry"
            role="doc-biblioentry">
            Botella, C., Gaüzère, P., O’Connor, L., Ohlmann, M., Renaud,
            J., Dou, Y., <em>et al.</em> (2023). <em><a
            href="https://doi.org/10.22541/au.168010211.10402875/v1">Land-use
            intensity influences European tetrapod food-webs</a></em>
            (Preprint). Authorea.
            </div>
            <div id="ref-Braga2019SpaAna" class="csl-entry"
            role="doc-biblioentry">
            Braga, J., Pollock, L.J., Barros, C., Galiana, N., Montoya,
            J.M., Gravel, D., <em>et al.</em> (2019). <a
            href="https://doi.org/10.1111/geb.12981">Spatial analyses of
            multi-trophic terrestrial vertebrate assemblages in
            Europe</a>. <em>Global Ecology and Biogeography</em>, 28,
            1636–1648.
            </div>
            <div id="ref-Cirtwill2015ConPre" class="csl-entry"
            role="doc-biblioentry">
            Cirtwill, A.R. &amp; Stouffer, D.B. (2015). <a
            href="https://doi.org/10.1111/1365-2656.12323">Concomitant
            predation on parasites is highly variable but constrains the
            ways in which parasites contribute to food web
            structure</a>. <em>Journal of Animal Ecology</em>, 84,
            734–744.
            </div>
            <div id="ref-daSilva2014LocReg" class="csl-entry"
            role="doc-biblioentry">
            da Silva, P.G. &amp; Hernández, M.I.M. (2014). <a
            href="https://doi.org/10.1371/journal.pone.0111883">Local
            and regional effects on community structure of dung beetles
            in a mainland-island scenario</a>. <em>PLOS ONE</em>, 9,
            e111883.
            </div>
            <div id="ref-Danisch2021MakJl" class="csl-entry"
            role="doc-biblioentry">
            Danisch, S. &amp; Krumbiegel, J. (2021). <a
            href="https://doi.org/10.21105/joss.03349">Makie.jl:
            Flexible high-performance data visualization for Julia</a>.
            <em>Journal of Open Source Software</em>, 6, 3349.
            </div>
            <div id="ref-Dansereau2022EvaEco" class="csl-entry"
            role="doc-biblioentry">
            Dansereau, G., Legendre, P. &amp; Poisot, T. (2022). <a
            href="https://doi.org/10.1111/oik.09063">Evaluating
            ecological uniqueness over broad spatial extents using
            species distribution modelling</a>. <em>Oikos</em>, 2022,
            e09063.
            </div>
            <div id="ref-Dansereau2021SimJl" class="csl-entry"
            role="doc-biblioentry">
            Dansereau, G. &amp; Poisot, T. (2021). <a
            href="https://doi.org/10.21105/joss.02872">SimpleSDMLayers.jl
            and GBIF.jl: A framework for species distribution modeling
            in Julia</a>. <em>Journal of Open Source Software</em>, 6,
            2872.
            </div>
            <div id="ref-Dansereau2023PoiSpa" class="csl-entry"
            role="doc-biblioentry">
            Dansereau, G. &amp; Poisot, T. (2023). <a
            href="https://doi.org/10.5281/ZENODO.8350065">PoisotLab/SpatialProbabilisticMetaweb:
            V1.0</a>.
            </div>
            <div id="ref-Desjardins-Proulx2017EcoInt" class="csl-entry"
            role="doc-biblioentry">
            Desjardins-Proulx, P., Laigle, I., Poisot, T. &amp; Gravel,
            D. (2017). <a
            href="https://doi.org/10.7717/peerj.3644">Ecological
            interactions and the Netflix problem</a>. <em>PeerJ</em>, 5,
            e3644.
            </div>
            <div id="ref-Dinerstein2017EcoApp" class="csl-entry"
            role="doc-biblioentry">
            Dinerstein, E., Olson, D., Joshi, A., Vynne, C., Burgess,
            N.D., Wikramanayake, E., <em>et al.</em> (2017). <a
            href="https://doi.org/10.1093/biosci/bix014">An
            Ecoregion-Based Approach to Protecting Half the Terrestrial
            Realm</a>. <em>BioScience</em>, 67, 534–545.
            </div>
            <div id="ref-Dunne2006NetStr" class="csl-entry"
            role="doc-biblioentry">
            Dunne, J. (2006). The network structure of food webs. In:
            <em>Ecological Networks: Linking Structure to Dynamics in
            Food Webs</em>. pp. 27–86.
            </div>
            <div id="ref-Frelat2022FooWeb" class="csl-entry"
            role="doc-biblioentry">
            Frelat, R., Kortsch, S., Kröncke, I., Neumann, H.,
            Nordström, M.C., Olivier, P.E.N., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1111/ecog.05945">Food web structure
            and community composition: A comparison across space and
            time in the North Sea</a>. <em>Ecography</em>, 2022.
            </div>
            <div id="ref-Galiana2021SpaSca" class="csl-entry"
            role="doc-biblioentry">
            Galiana, N., Barros, C., Braga, J., Ficetola, G.F.,
            Maiorano, L., Thuiller, W., <em>et al.</em> (2021). <a
            href="https://doi.org/10.1111/ecog.05229">The spatial
            scaling of food web structure across European
            biogeographical regions</a>. <em>Ecography</em>, 44,
            653–664.
            </div>
            <div id="ref-Galiana2022EcoNet" class="csl-entry"
            role="doc-biblioentry">
            Galiana, N., Lurgi, M., Bastazini, V.A.G., Bosch, J.,
            Cagnolo, L., Cazelles, K., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1038/s41559-021-01644-4">Ecological
            network complexity scales with area</a>. <em>Nature Ecology
            &amp; Evolution</em>, 1–8.
            </div>
            <div id="ref-Gauzere2022DivBio" class="csl-entry"
            role="doc-biblioentry">
            Gaüzère, P., O’Connor, L., Botella, C., Poggiato, G.,
            Münkemüller, T., Pollock, L.J., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1016/j.cub.2022.03.009">The
            diversity of biotic interactions complements functional and
            phylogenetic facets of biodiversity</a>. <em>Current
            Biology</em>.
            </div>
            <div id="ref-GBIFSecretariat2021GbiBac" class="csl-entry"
            role="doc-biblioentry">
            GBIF Secretariat. (2021). <a
            href="https://doi.org/10.15468/39omei">GBIF Backbone
            Taxonomy</a>.
            </div>
            <div id="ref-GBIF.org2022GbiOcc" class="csl-entry"
            role="doc-biblioentry">
            GBIF.org. (2022). <a
            href="https://doi.org/10.15468/DL.H7GZPV">GBIF occurrence
            download</a>.
            </div>
            <div id="ref-GDAL/OGRcontributors2021GdaOgr"
            class="csl-entry" role="doc-biblioentry">
            GDAL/OGR contributors. (2021). <em>GDAL/OGR geospatial data
            abstraction software library</em>. Manual. Open Source
            Geospatial Foundation.
            </div>
            <div id="ref-Gravel2019BriElt" class="csl-entry"
            role="doc-biblioentry">
            Gravel, D., Baiser, B., Dunne, J.A., Kopelke, J.-P.,
            Martinez, N.D., Nyman, T., <em>et al.</em> (2019). <a
            href="https://doi.org/10.1111/ecog.04006">Bringing Elton and
            Grinnell together: A quantitative framework to represent the
            biogeography of ecological interaction networks</a>.
            <em>Ecography</em>, 42, 401–415.
            </div>
            <div id="ref-Guisan2005PreSpe" class="csl-entry"
            role="doc-biblioentry">
            Guisan, A. &amp; Thuiller, W. (2005). <a
            href="https://doi.org/10.1111/j.1461-0248.2005.00792.x">Predicting
            species distribution: Offering more than simple habitat
            models</a>. <em>Ecology Letters</em>, 8, 993–1009.
            </div>
            <div id="ref-Heino2017ExpSpe" class="csl-entry"
            role="doc-biblioentry">
            Heino, J. &amp; Grönroos, M. (2017). <a
            href="https://doi.org/10.1007/s00442-016-3754-7">Exploring
            species and site contributions to beta diversity in stream
            insect assemblages</a>. <em>Oecologia</em>, 183, 151–160.
            </div>
            <div id="ref-Hortal2015SevSho" class="csl-entry"
            role="doc-biblioentry">
            Hortal, J., de Bello, F., Diniz-Filho, J.A.F., Lewinsohn,
            T.M., Lobo, J.M. &amp; Ladle, R.J. (2015). <a
            href="https://doi.org/10.1146/annurev-ecolsys-112414-054400">Seven
            Shortfalls that Beset Large-Scale Knowledge of
            Biodiversity</a>. <em>Annual Review of Ecology, Evolution,
            and Systematics</em>, 46, 523–549.
            </div>
            <div id="ref-Jordano2016SamNet" class="csl-entry"
            role="doc-biblioentry">
            Jordano, P. (2016). <a
            href="https://doi.org/10.1111/1365-2435.12763">Sampling
            networks of ecological interactions</a>. <em>Functional
            Ecology</em>, 30, 1883–1893.
            </div>
            <div id="ref-Karger2017CliHig" class="csl-entry"
            role="doc-biblioentry">
            Karger, D.N., Conrad, O., Böhner, J., Kawohl, T., Kreft, H.,
            Soria-Auza, R.W., <em>et al.</em> (2017). <a
            href="https://doi.org/10.1038/sdata.2017.122">Climatologies
            at high resolution for the earth’s land surface areas</a>.
            <em>Scientific Data</em>, 4, 170122.
            </div>
            <div id="ref-Karger2018DatCli" class="csl-entry"
            role="doc-biblioentry">
            Karger, D.N., Conrad, O., Böhner, J., Kawohl, T., Kreft, H.,
            Soria-Auza, R.W., <em>et al.</em> (2018). <a
            href="https://doi.org/10.5061/DRYAD.KD1D4">Data from:
            Climatologies at high resolution for the earth’s land
            surface areas</a>.
            </div>
            <div id="ref-Karger2021CliHig" class="csl-entry"
            role="doc-biblioentry">
            Karger, D.N., Conrad, O., Böhner, J., Kawohl, T., Kreft, H.,
            Soria-Auza, R.W., <em>et al.</em> (2021). <a
            href="https://doi.org/10.16904/ENVIDAT.228">Climatologies at
            high resolution for the earth’s land surface areas</a>.
            </div>
            <div id="ref-Kortsch2021DisTem" class="csl-entry"
            role="doc-biblioentry">
            Kortsch, S., Frelat, R., Pecuchet, L., Olivier, P., Putnis,
            I., Bonsdorff, E., <em>et al.</em> (2021). <a
            href="https://doi.org/10.1111/1365-2656.13447">Disentangling
            temporal food web dynamics facilitates understanding of
            ecosystem functioning</a>. <em>Journal of Animal
            Ecology</em>, 90, 1205–1216.
            </div>
            <div id="ref-Kortsch2019FooStr" class="csl-entry"
            role="doc-biblioentry">
            Kortsch, S., Primicerio, R., Aschan, M., Lind, S., Dolgov,
            A.V. &amp; Planque, B. (2019). <a
            href="https://doi.org/10.1111/ecog.03443">Food-web structure
            varies along environmental gradients in a high-latitude
            marine ecosystem</a>. <em>Ecography</em>, 42, 295–308.
            </div>
            <div id="ref-Kortsch2015CliCha" class="csl-entry"
            role="doc-biblioentry">
            Kortsch, S., Primicerio, R., Fossheim, M., Dolgov, A.V.
            &amp; Aschan, M. (2015). <a
            href="https://doi.org/10.1098/rspb.2015.1546">Climate change
            alters the structure of arctic marine food webs due to
            poleward shifts of boreal generalists</a>. <em>Proceedings
            of the Royal Society B: Biological Sciences</em>, 282,
            20151546.
            </div>
            <div id="ref-Legendre2013BetDiv" class="csl-entry"
            role="doc-biblioentry">
            Legendre, P. &amp; De Cáceres, M. (2013). <a
            href="https://doi.org/10.1111/ele.12141">Beta diversity as
            the variance of community data: Dissimilarity coefficients
            and partitioning</a>. <em>Ecology Letters</em>, 16, 951–963.
            </div>
            <div id="ref-Lopez-Lopez2022EcoNet" class="csl-entry"
            role="doc-biblioentry">
            López-López, L., Genner, M.J., Tarling, G.A., Saunders, R.A.
            &amp; O’Gorman, E.J. (2022). <a
            href="https://doi.org/10.1007/s10021-021-00665-1">Ecological
            Networks in the Scotia Sea: Structural Changes Across
            Latitude and Depth</a>. <em>Ecosystems</em>, 25, 457–470.
            </div>
            <div id="ref-Lucas2023IncBio" class="csl-entry"
            role="doc-biblioentry">
            Lucas, P., Thuiller, W., Talluto, M., Polaina, E., Albrecht,
            J., Selva, N., <em>et al.</em> (2023). <a
            href="https://doi.org/10.1101/2023.03.10.532098">Including
            biotic interactions in species distribution models improves
            the understanding of species niche: A case of study with the
            brown bear in Europe</a>.
            </div>
            <div id="ref-Lurgi2020GeoVar" class="csl-entry"
            role="doc-biblioentry">
            Lurgi, M., Galiana, N., Broitman, B.R., Kéfi, S., Wieters,
            E.A. &amp; Navarrete, S.A. (2020). <a
            href="https://doi.org/10.1002/ecy.3165">Geographical
            variation of multiplex ecological networks in marine
            intertidal communities</a>. <em>Ecology</em>, 101, e03165.
            </div>
            <div id="ref-Maiorano2020TetSpe" class="csl-entry"
            role="doc-biblioentry">
            Maiorano, L., Montemaggiori, A., Ficetola, G.F., O’Connor,
            L. &amp; Thuiller, W. (2020). <a
            href="https://doi.org/10.1111/geb.13138">TETRA-EU 1.0: A
            species-level trophic metaweb of European tetrapods</a>.
            <em>Global Ecology and Biogeography</em>, 29, 1452–1457.
            </div>
            <div id="ref-Martins2022GloReg" class="csl-entry"
            role="doc-biblioentry">
            Martins, L.P., Stouffer, D.B., Blendinger, P.G.,
            Böhning-Gaese, K., Buitrón-Jurado, G., Correia, M., <em>et
            al.</em> (2022). <a
            href="https://doi.org/10.1038/s41467-022-34355-w">Global and
            regional ecological boundaries explain abrupt spatial
            discontinuities in avian frugivory interactions</a>.
            <em>Nature Communications</em>, 13, 6943.
            </div>
            <div id="ref-McElreath2020StaRet" class="csl-entry"
            role="doc-biblioentry">
            McElreath, R. (2020). <em><a
            href="https://doi.org/10.1201/9780429029608">Statistical
            rethinking: A bayesian course with examples in R and
            Stan</a></em>. Second. Chapman and Hall/CRC, New York.
            </div>
            <div id="ref-McLeod2021SamAsy" class="csl-entry"
            role="doc-biblioentry">
            McLeod, A., Leroux, S.J., Gravel, D., Chu, C., Cirtwill,
            A.R., Fortin, M.-J., <em>et al.</em> (2021). <a
            href="https://doi.org/10.1111/oik.08650">Sampling and
            asymptotic network properties of spatial multi-trophic
            networks</a>. <em>Oikos</em>, 130, 2250–2259.
            </div>
            <div id="ref-Mestre2022DisFoo" class="csl-entry"
            role="doc-biblioentry">
            Mestre, F., Gravel, D., García-Callejas, D., Pinto-Cruz, C.,
            Matias, M.G. &amp; Araújo, M.B. (2022). <a
            href="https://doi.org/10.1016/j.baae.2022.03.011">Disentangling
            food-web environment relationships: A review with
            guidelines</a>. <em>Basic and Applied Ecology</em>, 61,
            102–115.
            </div>
            <div id="ref-Milo2002NetMot" class="csl-entry"
            role="doc-biblioentry">
            Milo, R., Shen-Orr, S., Itzkovitz, S., Kashtan, N.,
            Chklovskii, D. &amp; Alon, U. (2002). <a
            href="https://doi.org/10.1126/science.298.5594.824">Network
            Motifs: Simple Building Blocks of Complex Networks</a>.
            <em>Science</em>, 298, 824–827.
            </div>
            <div id="ref-Moens2022ImpBio" class="csl-entry"
            role="doc-biblioentry">
            Moens, M., Biesmeijer, J., Huang, E., Vereecken, N. &amp;
            Marshall, L. (2022). <a
            href="https://doi.org/10.22541/au.166784603.30222525/v1">The
            importance of biotic interactions in distribution models
            depends on the type of ecological relations, spatial scale
            and range</a>.
            </div>
            <div id="ref-Mora2018IdeCom" class="csl-entry"
            role="doc-biblioentry">
            Mora, B.B., Gravel, D., Gilarranz, L.J., Poisot, T. &amp;
            Stouffer, D.B. (2018). <a
            href="https://doi.org/10.1038/s41467-018-05056-0">Identifying
            a common backbone of interactions underlying food webs from
            different ecosystems</a>. <em>Nature Communications</em>, 9,
            2603.
            </div>
            <div id="ref-Morales-Castilla2015InfBio" class="csl-entry"
            role="doc-biblioentry">
            Morales-Castilla, I., Matias, M.G., Gravel, D. &amp; Araújo,
            M.B. (2015). <a
            href="https://doi.org/10.1016/j.tree.2015.03.014">Inferring
            biotic interactions from proxies</a>. <em>Trends in Ecology
            &amp; Evolution</em>, 30, 347–356.
            </div>
            <div id="ref-OConnor2020UnvFoo" class="csl-entry"
            role="doc-biblioentry">
            O’Connor, L.M.J., Pollock, L.J., Braga, J., Ficetola, G.F.,
            Maiorano, L., Martinez-Almoyna, C., <em>et al.</em> (2020).
            <a href="https://doi.org/10.1111/jbi.13773">Unveiling the
            food webs of tetrapods across Europe through the prism of
            the Eltonian niche</a>. <em>Journal of Biogeography</em>,
            47, 181–192.
            </div>
            <div id="ref-Olivier2019ExpTem" class="csl-entry"
            role="doc-biblioentry">
            Olivier, P., Frelat, R., Bonsdorff, E., Kortsch, S.,
            Kröncke, I., Möllmann, C., <em>et al.</em> (2019). <a
            href="https://doi.org/10.1111/ecog.04461">Exploring the
            temporal variability of a food web using long-term
            biomonitoring data</a>. <em>Ecography</em>, 42, 2107–2121.
            </div>
            <div id="ref-Pecuchet2020NovFee" class="csl-entry"
            role="doc-biblioentry">
            Pecuchet, L., Blanchet, M.-A., Frainer, A., Husson, B.,
            Jørgensen, L.L., Kortsch, S., <em>et al.</em> (2020). <a
            href="https://doi.org/10.1111/gcb.15196">Novel feeding
            interactions amplify the impact of species redistribution on
            an Arctic food web</a>. <em>Global Change Biology</em>, 26,
            4894–4906.
            </div>
            <div id="ref-Poggiato2022IntFoo" class="csl-entry"
            role="doc-biblioentry">
            Poggiato, G., Andréoletti, J., Shirley, L. &amp; Thuiller,
            W. (2022). <em><a
            href="https://doi.org/10.22541/au.166853394.45823739/v1">Integrating
            food webs in species distribution models improves ecological
            niche estimation and predictions</a></em> (Preprint).
            Authorea.
            </div>
            <div id="ref-Poisot2019EcoJl" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Bélisle, Z., Hoebeke, L., Stock, M. &amp;
            Szefer, P. (2019). <a
            href="https://doi.org/10.1111/ecog.04310">EcologicalNetworks.jl:
            Analysing ecological networks of species interactions</a>.
            <em>Ecography</em>, 42, 1850–1861.
            </div>
            <div id="ref-Poisot2021GloKno" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Bergeron, G., Cazelles, K., Dallas, T., Gravel,
            D., MacDonald, A., <em>et al.</em> (2021). <a
            href="https://doi.org/10.1111/jbi.14127">Global knowledge
            gaps in species interaction networks data</a>. <em>Journal
            of Biogeography</em>, 48, 1552–1563.
            </div>
            <div id="ref-Poisot2012DisSpe" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Canard, E., Mouillot, D., Mouquet, N. &amp;
            Gravel, D. (2012). <a
            href="https://doi.org/10.1111/ele.12002">The dissimilarity
            of species interaction networks</a>. <em>Ecology
            Letters</em>, 15, 1353–1361.
            </div>
            <div id="ref-Poisot2016StrPro" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Cirtwill, A.R., Cazelles, K., Gravel, D.,
            Fortin, M.-J. &amp; Stouffer, D.B. (2016). <a
            href="https://doi.org/10.1111/2041-210X.12468">The structure
            of probabilistic networks</a>. <em>Methods in Ecology and
            Evolution</em>, 7, 303–312.
            </div>
            <div id="ref-Poisot2017HosPar" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Guéveneux-Julien, C., Fortin, M.-J., Gravel, D.
            &amp; Legendre, P. (2017). <a
            href="https://doi.org/10.1111/geb.12602">Hosts, parasites
            and their interactions respond to different climatic
            variables</a>. <em>Global Ecology and Biogeography</em>, 26,
            942–951.
            </div>
            <div id="ref-Poisot2023NetEmb" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Ouellet, M.-A., Mollentze, N., Farrell, M.J.,
            Becker, D.J., Brierley, L., <em>et al.</em> (2023). <a
            href="https://doi.org/10.1016/j.patter.2023.100738">Network
            embedding unveils the hidden interactions in the mammalian
            virome</a>. <em>Patterns</em>, 4, 100738.
            </div>
            <div id="ref-Poisot2015SpeWhy" class="csl-entry"
            role="doc-biblioentry">
            Poisot, T., Stouffer, D.B. &amp; Gravel, D. (2015). <a
            href="https://doi.org/10.1111/oik.01719">Beyond species: Why
            ecological interaction networks vary through space and
            time</a>. <em>Oikos</em>, 124, 243–251.
            </div>
            <div id="ref-Saravia2022EcoNet" class="csl-entry"
            role="doc-biblioentry">
            Saravia, L.A., Marina, T.I., Kristensen, N.P., De Troch, M.
            &amp; Momo, F.R. (2022). <a
            href="https://doi.org/10.1111/1365-2656.13652">Ecological
            network assembly: How the regional metaweb influences local
            food webs</a>. <em>Journal of Animal Ecology</em>, 91,
            630–642.
            </div>
            <div id="ref-Simmons2019MotBip" class="csl-entry"
            role="doc-biblioentry">
            Simmons, B.I., Cirtwill, A.R., Baker, N.J., Wauchope, H.S.,
            Dicks, L.V., Stouffer, D.B., <em>et al.</em> (2019). <a
            href="https://doi.org/10.1111/oik.05670">Motifs in bipartite
            ecological networks: Uncovering indirect interactions</a>.
            <em>Oikos</em>, 128, 154–170.
            </div>
            <div id="ref-StatisticsCanada2022BouFil" class="csl-entry"
            role="doc-biblioentry">
            Statistics Canada. (2022). <em>Boundary files, reference
            guide second edition, Census year 2021</em>. Second edition.
            Statistics Canada = Statistique Canada, Ottawa.
            </div>
            <div id="ref-Stouffer2010UndFoo" class="csl-entry"
            role="doc-biblioentry">
            Stouffer, D.B. &amp; Bascompte, J. (2010). <a
            href="https://doi.org/10.1111/j.1461-0248.2009.01407.x">Understanding
            food-web persistence from local to global scales</a>.
            <em>Ecology Letters</em>, 13, 154–161.
            </div>
            <div id="ref-Stouffer2007EviExi" class="csl-entry"
            role="doc-biblioentry">
            Stouffer, D.B., Camacho, J., Jiang, W. &amp; Nunes Amaral,
            L.A. (2007). <a
            href="https://doi.org/10.1098/rspb.2007.0571">Evidence for
            the existence of a robust pattern of prey selection in food
            webs</a>. <em>Proceedings of the Royal Society B: Biological
            Sciences</em>, 274, 1931–1940.
            </div>
            <div id="ref-Strydom2022FooWeb" class="csl-entry"
            role="doc-biblioentry">
            Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron,
            D., Farrell, M.J., <em>et al.</em> (2022). <a
            href="https://doi.org/10.1111/2041-210X.13835">Food web
            reconstruction through phylogenetic transfer of low-rank
            network representation</a>. <em>Methods in Ecology and
            Evolution</em>, 13, 2838–2849.
            </div>
            <div id="ref-Strydom2023GraEmb" class="csl-entry"
            role="doc-biblioentry">
            Strydom, T., Bouskila, S., Banville, F., Barros, C., Caron,
            D., Farrell, M.J., <em>et al.</em> (2023). <a
            href="https://doi.org/10.1111/2041-210X.14228">Graph
            embedding and transfer learning can help predict potential
            species interaction networks despite data limitations</a>.
            <em>Methods in Ecology and Evolution</em>, 14, 2917–2930.
            </div>
            <div id="ref-Strydom2021RoaPre" class="csl-entry"
            role="doc-biblioentry">
            Strydom, T., Catchen, M.D., Banville, F., Caron, D.,
            Dansereau, G., Desjardins-Proulx, P., <em>et al.</em>
            (2021). <a href="https://doi.org/10.1098/rstb.2021.0063">A
            roadmap towards predicting species interaction networks
            (across space and time)</a>. <em>Philosophical Transactions
            of the Royal Society B: Biological Sciences</em>, 376,
            20210063.
            </div>
            <div id="ref-Tuanmu2014Glo1km" class="csl-entry"
            role="doc-biblioentry">
            Tuanmu, M.-N. &amp; Jetz, W. (2014). <a
            href="https://doi.org/10.1111/geb.12182">A global 1-km
            consensus land-cover product for biodiversity and ecosystem
            modelling</a>. <em>Global Ecology and Biogeography</em>, 23,
            1031–1045.
            </div>
            <div id="ref-Vasconcelos2018ExpImp" class="csl-entry"
            role="doc-biblioentry">
            Vasconcelos, T.S., Nascimento, B.T.M. do &amp; Prado, V.H.M.
            (2018). <a href="https://doi.org/10.1002/ece3.4357">Expected
            impacts of climate change threaten the anuran diversity in
            the Brazilian hotspots</a>. <em>Ecology and Evolution</em>,
            8, 7894–7906.
            </div>
            <div id="ref-Windsor2023UsiEco" class="csl-entry"
            role="doc-biblioentry">
            Windsor, F.M., van den Hoogen, J., Crowther, T.W. &amp;
            Evans, D.M. (2023). <a
            href="https://doi.org/10.1111/jbi.14447">Using ecological
            networks to answer questions in global biogeography and
            ecology</a>. <em>Journal of Biogeography</em>, 50, 57–69.
            </div>
            <div id="ref-Wisz2013RolBio" class="csl-entry"
            role="doc-biblioentry">
            Wisz, M.S., Pottier, J., Kissling, W.D., Pellissier, L.,
            Lenoir, J., Damgaard, C.F., <em>et al.</em> (2013). <a
            href="https://doi.org/10.1111/j.1469-185X.2012.00235.x">The
            role of biotic interactions in shaping distributions and
            realised assemblages of species: Implications for species
            distribution modelling</a>. <em>Biological Reviews</em>, 88,
            15–30.
            </div>
            <div id="ref-Youden1950IndRat" class="csl-entry"
            role="doc-biblioentry">
            Youden, W.J. (1950). <a
            href="https://doi.org/10.1002/1097-0142(1950)3:1&lt;32::AID-CNCR2820030106&gt;3.0.CO;2-3">Index
            for rating diagnostic tests</a>. <em>Cancer</em>, 3, 32–35.
            </div>
            <div id="ref-Zarnetske2017IntLan" class="csl-entry"
            role="doc-biblioentry">
            Zarnetske, P.L., Baiser, B., Strecker, A., Record, S.,
            Belmaker, J. &amp; Tuanmu, M.-N. (2017). <a
            href="https://doi.org/10.1007/s40823-017-0021-5">The
            Interplay Between Landscape Structure and Biotic
            Interactions</a>. <em>Current Landscape Ecology
            Reports</em>, 2, 12–29.
            </div>
            </div>
        </article>
    </main>

    <footer>
                <div class="container">
            <h2>Affiliations</h2>
            <sup>1</sup>&nbsp;Département de sciences biologiques,
            Université de Montréal, Montréal, QC,
            Canada; <sup>2</sup>&nbsp;Quebec Centre for Biodiversity
            Science, Montréal, QC, Canada; <sup>3</sup>&nbsp;Department
            of Forest Resources Management, University of British
            Columbia, Vancouver, BC, Canada
            <br />
            
        </div>
    </footer>

</body>

</html>